freebayes/1.3.6/Dockerfile

```diff
@@ -0,0 +1,86 @@
+FROM ubuntu:focal as app
```

Comment:
Seriously, though, look at how beautiful and clear this dockerfile is. 

I'm so happy you're contributing to this repo.